src/common/dsp/DPWOscillator.cpp

```diff
@@ -137,23 +138,23 @@ void DPWOscillator::process_block(float pitch, float drift, bool stereo, bool FM
 
 void DPWOscillator::init_ctrltypes()
 {
-    oscdata->p[dpw_saw_mix].set_name("Saw Mix");
+    oscdata->p[dpw_saw_mix].set_name("Sawtooth");
     oscdata->p[dpw_saw_mix].set_type(ct_percent);
-    oscdata->p[dpw_saw_mix].val_default.f = 0.75;
+    oscdata->p[dpw_saw_mix].val_default.f = 0.5;
 
-    oscdata->p[dpw_pulse_mix].set_name("Pulse Mix");
+    oscdata->p[dpw_pulse_mix].set_name("Pulse");
     oscdata->p[dpw_pulse_mix].set_type(ct_percent);
-    oscdata->p[dpw_pulse_mix].val_default.f = 0.75;
+    oscdata->p[dpw_pulse_mix].val_default.f = 0.5;
 
-    oscdata->p[dpw_tri_mix].set_name("Triangle Mix");
+    oscdata->p[dpw_tri_mix].set_name("Triangle");
     oscdata->p[dpw_tri_mix].set_type(ct_percent);
-    oscdata->p[dpw_tri_mix].val_default.f = 0.75;
+    oscdata->p[dpw_tri_mix].val_default.f = 0.5;
 
-    oscdata->p[dpw_pulse_width].set_name("Pulse Width");
+    oscdata->p[dpw_pulse_width].set_name("Width");
     oscdata->p[dpw_pulse_width].set_type(ct_percent);
     oscdata->p[dpw_pulse_width].val_default.f = 0.5;
 
-    oscdata->p[dpw_sync].set_name("Currently Unimplemented");
```

Comment:
Can ou please not make this change until I have sync in place? I did it on purpose to avoid nightly testers reporting that sync didn't work!